tawazi/consts.py

```diff
@@ -90,12 +95,10 @@ def __deepcopy__(self, _prev: Dict[Any, Any]) -> "NoValType":
 Identifier = str
 Tag = Union[None, str, tuple]  # anything immutable
 
-# NOTE: maybe support other key types? for example int... or even tuple...
-ReturnIDsType = Optional[
-    Union[Dict[str, Identifier], List[Identifier], Tuple[Identifier, ...], Identifier]
-]
 RVTypes = Union[Any, Tuple[Any], List[Any], Dict[str, Any]]
 P = ParamSpec("P")
 RVDAG = TypeVar("RVDAG", bound=RVTypes, covariant=True)
 
 RVXN = TypeVar("RVXN", covariant=True)
+
+# ImmutableType = Union[str, int, float, bool, Tuple[ImmutableType]]  # doesn't work because of cyclic typing
```

Comment:
should we remove that if it's not used ?

Reply:
this is here as a reminder actually

Reply:
I will transform this into a TODO